third-party/components/ApprovalCommentForm/ApprovalCommentForm.js

```diff
@@ -10,7 +10,7 @@ import classNames from 'classnames';
 import { FormattedMessage, injectIntl } from 'react-intl';
 
 import Form from 'box-react-ui/lib/components/form-elements/form/Form';
-import TextArea from 'box-react-ui/lib/components/text-area/TextArea';
+import TextArea from 'box-react-ui/lib/components/form-elements/text-area/TextArea';
```

Comment:
This TextArea accepts a validation method as a param and handles triggering that validation when the textarea is empty, too long, etc.